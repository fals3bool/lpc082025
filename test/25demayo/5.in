2
Arroz 0.20
Pollo 0.50
2 4
Arroz 0.50
Pollo 1.00
