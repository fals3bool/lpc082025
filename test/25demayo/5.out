Arroz 0.3
Pollo 1.0
